### DISCUSSION TITLE
Help Analyzing Movement in Drosophila Forced Swim Test

### DISCUSSION THREAD

**Discussion by ebmiller74** [talmolab/sleap#1702] [Help!]
I am analyzing the movement of Drosophila in a forced swim test and I want to compare the distance between the thorax and each of its legs to determine if the flies are actually moving or simply drifting in the water. Can this be done with sleap?

**Comment by Marielamend** [comment]
Hello @ebmiller74!

You can [export your data for analysis](https://sleap.ai/tutorials/analysis.html) so you can pull out the coordinates and compute distances and angles. [See here](https://sleap.ai/notebooks/Analysis_examples.html) for some examples. You can also use [DeepOF](https://deepof.readthedocs.io/en/latest/#) to generate coordinates and distances from your data! 

Let us know if you have any other questions!
-Mariela
